spec/features/survey_response_new_survey_spec.rb

```diff
@@ -0,0 +1,74 @@
+require "rails_helper"
+
+feature "Submitting a new code confidence survey", js: true do
+  scenario "A user can enter in a number for perceived code confidence" do
+    sign_in
+
+    visit new_survey_response_path
+
+    expect(page).to have_content("How confident am I")
+
+    select "5", :from => "How confident am I"
+
+    expect(page).to have_content("5")
```

Comment:
I think this test could probably be combined with the test below it to exercise a bit more code than it currently does. Just making sure that selecting an item doesn't really test your code out, since it's the built in Rails helpers that are responsible for making sure the select works.

It might be nice to combine this test with the other. Select both of the options, then submit the form and verify that the SurveyResponse was actually created, either through the UI or directly checking in the database if you need to.